decode_test.go

```diff
@@ -513,3 +514,52 @@ func TestDecodeTagSkip(t *testing.T) {
 		t.Error("field decoded when it was tagged as -")
 	}
 }
+
+// test parity with plutil -lint on macOS
+var xmlParityTestFailures = map[string]struct{}{
```

Comment:
nit: I don't like the magic dictionary here. What if you encoded the plist testability within the filename? For example `empty-plist.plist` -> `empty-plist.plutil.plist`?

again, non-blocker. :)

Reply:
@jessepeterson I added a commit with this suggestion. Can you re-review `decode_test.go: TestXMLPlutilParity`?